### ISSUE TITLE
Request: Unblock app regional restrictions

### ISSUE THREAD

**Issue by etis** [boxeehacks/boxeehack#505]
Boxee limits the access to regional restricted apps like iPlayer and Netflix on the app list unless you’re in that region or connected through a VPN server.
In older version there used to be a setting to show or hide these apps (even though showing them didn't allow access without a VPN), but it's gone on the latest version.

There are some new solutions on the market that use DNS server instead of VPN for faster connection. They work great if you use the browser to access the content, but Boxee hide these apps unless your IP is a US or UK.

I found a way to hack it and access the apps with Boxee+, and configured my TV shortcut as iPlayer and Movies as Netflix, but I want to be able to access the regular TV and Movies menus as well.
Can you please add new shortcuts on the home screen for an app or two or make the shortcut changes more flexible (so I can change the music shortcut, that I don't use, to Netflix for example)? Or hack the app restriction so all apps are available to install?

**Comment by ducapedia** [comment]
Netflix was offically here in Brazil at least 1 year.
Works fine without DNSmasq, VPN or anykind of httptunnel, in PS3, Xbox, SmarTVs, smartphones, tablets (android and iOS), PC etc. ... any gadgets but Boxeebox.
The offical explain to this (by Dlink and Neflix), is the BoxeeBox is not licensed by netflix to work in Brazil (althounght its officially here).
When I try to use these kind of hacks (VPN, etc), Netflix app shortcut apperas in boxee dash, but opens US (or whenever was VPN to) content.. not Brazilian content.
I dont think they was blocking by IP region, because was I sad... they are officialy here.. In Boxee setup, in service I see a kind of a serial number for Netflix Service, ...  i supose thats the key for this particular case.

**Comment by egcarrano** [comment]
If I am understanding it correctly, you Etis was able to access the netflix app on boxee outside US or UK. Please explain us how to do that.

**Comment by ducapedia** [comment]
Yeah.. I would like to know too.. How do you do to access netflix app, if it does not show up in app list ?

**Comment by etis** [comment]
@sebvalmontx360 @effduca There are tons of guides for this on the web. http://bit.ly/Z1lH0t
I wrote one myself but it's not in English. You can try using it using Google Translate though http://goo.gl/8u0gj

 Basically you need to buy (yes buy, there's really no good free alternatives) a VPN connection.Then configure it on your BB, it's easy. Once you're connected to a US VPN server you need to register to Netflix using a (fake) US address, and pay with any international credit card. You do it from your laptop, not BB. I've been doing this for a couple years now and i'm a happy costumer. 

As for a recommended VPN provider, I know unblock-us is quite popular 
I'm a happy costumer of ibvpn.com, that also provides the new DNS solution. It's new and still in beta, and I had trouble using it for US services on the BB but it's pretty awesome. I can access Netflix, Hulu and iPlayer from my laptop without needing to connect to a VPN server, or have annoying speed limitations that hurt the streaming. I also use it to watch BBC and ITV live, in HD on the BB. The streaming is perfect.

**Comment by ducapedia** [comment]
Oh.. you just used a VPN network. We already used this. But we thinking you was found some kind of hack or bug to show up Netflix app withou masking your IP to Netflix server.
VPN is not usefull, because as I sad, Netflix already is officially in Brazil, and only not working in boxeebox. With VPN we are able to access anothers Netflix servicies (anothers countries, without portuguese subtitles and anothers contents).
But, thanks anyway.

**Comment by etis** [comment]
@effduca Try changing Boxee+Hacks home screen settings so it shows Netflix button instead of Movies.
Go to settings > Boxee+hacks > Home Screen > Choose Friends or Movies and change the setting to Netflix.

I'm not sure this will work since I believe Boxee needs to create a seperate app for other regions, but it worth a try.

**Comment by ducapedia** [comment]
Definitly worth a try.
Thanks for the hint... I'll let you know if works when I try.

**Comment by ducapedia** [comment]
Put Netflix as option in HomeScreen Settings by Boxee+Hacks definitly allows me launch Netflix app, even without it shows in apps tab.
But, when I start it, its open.. logs withou any errors, and then shows a message "Its not possible acess netflix server by now. Try again later". I tried at same time in my smartphone and I acess without any problems.
I see another ticket issue here from people claimed for this error even where was working before the last boxee hack update.
So, by now, I can say if this error means thats your hint didnt work.
In time, I guess I can put netflix shortcut in HomeScreen icon called "Friend" and by it acess content, as is we dont miss the "Movies" shortcut.

**Comment by prodriguezii** [comment]
See the following ticket in the Boxer bug tracker

http://jira.boxee.tv/browse/BOXEE-11389

Sent from my Windows Phone

**Comment by ducapedia** [comment]
I see... I'll let boxee under tunlr DNS and watch only US contents.. Its working by now. If I wanna see BR contents, I'll need to turn on Xbox or PS3.

**Comment by kvz** [comment]
https://github.com/boxeehacks/boxeehack/issues/432#issuecomment-24948481
